packages/components/RadioGroup/src/RadioButton.types.ts

```diff
@@ -47,6 +47,12 @@ export interface IRadioButtonProps extends IPressableProps {
    */
   ariaSetSize?: number;
 
+  // Defined in IViewWin32, and duplicated here because RadioButton has already shipped supporting these properties. Rather than
+  // remove them from non-Windows platforms, they are being added here to align the types across all platforms. TypeScript will
+  // merge the prop definitions without complaint.
+  accessibilityPositionInSet?: number;
+  accessibilitySetSize?: number;
+
```

Comment:
Do we not inherit these from IPressableProps on line 12? My impression was IPressableProps should inherit from IViewProps (from `@fluentui-react-native/adapters`, which should have the right props per platform).